bot/bot.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from aiogram import Bot, Dispatcher, executor, types
+
+logging.basicConfig(level=logging.INFO)
+
+bot = Bot(token = '?')
```

Comment:
**PEP 8: E251 unexpected spaces around keyword / parameter equals**
What does it mean? Read [here](https://www.flake8rules.com/rules/E251.html)
In code it's 7:16